### PR TITLE
feat: add support for Edge Chromium browser

### PR DESCRIPTION
**Pull request by Ldoppea**
Add support for Edge Chromium. This implies the ability to go beyond the first `#installation` page (user were blocked by a `Not available` message), but also all instructions for installing the correct extension on the last step.

An Edge-Chromium icon as been added to cozy-ui : https://github.com/cozy/cozy-ui/pull/1828

~~Package.json still has to be updated (so this PR is draft)~~

#### Checklist

Before merging this PR, the following things must have been done:

- [x] Faithful integration of the mockups at all screen sizes
- [x] Tested on supported browsers, including responsive mode (Edge, Chrome >= 42, 2 latest versions of Firefox and Safari)
- [ ] Localized in english and french
- [x] All changes have test coverage
- [ ] Updated README, if necessary
- [x] Make sure commit messages rely on the [Commitizen](http://commitizen.github.io/cz-cli/) format

If this PR relies on changes in one of our internal libraries (cozy-client or cozy-ui, for example), please also do the following things:

- [ ] The package.json and yarn.lock files include the correct dependency version, in those versions have been released
- [ ] [cozy-client](https://github.com/cozy/cozy-client-js) documentation and tests are up to date
- [x] [cozy-ui](https://github.com/cozy/cozy-ui) documentation and style guides are up to date

### CODE REVIEW

**Review by Crash-- — APPROVED** [review]

**Comment by Ldoppea** [comment]
Cozy-UI has been merged and `package.json` now reflects the change. PR isn't draft anymore.

**Comment by Crash--** [comment]
Since the PR was already approved, you can merge it without having a second review ;)

**Pull request merged**
